lib/rx/client.rb

```diff
@@ -66,6 +73,11 @@ def post_preferences(params)
 
     private
 
+    def cache_key(action)
+      return nil unless config.caching_enabled?
+      "#{session.user_id}:#{action}"
```

Comment:
is there or could there ever be a possibility that the `session.user_id would be nil?

Reply:
user_id in this context is the mhv_correlation_id, so it should never be nil if session is valid, but I can have it return nil if that's the case as precaution.